pandas/io/parsers/base_parser.py

```diff
@@ -556,6 +564,12 @@ def _convert_to_ndarrays(
             else:
                 col_na_values, col_na_fvalues = set(), set()
 
+            if c in self._parse_date_cols:
```

Comment:
comment here on why this is would be helpful

Reply:
Done